public_html/templates/base.html

```diff
@@ -75,25 +77,39 @@
 
 	{% block main %}
 		<section class="filters container-fluid">
-			<form>
-				<fieldset class="form-group filters-fieldset text-center">
-					{% for type, description in filterTypes %}
-						<span class="radio-inline">
+			<form id="filters-form">
+				<fieldset class="form-group filters-fieldset col-lg-6">
```

Comment:
Centered looks much better! 
Here's what I see on my phone:
![img_8424](https://cloud.githubusercontent.com/assets/5735280/17011650/935c9fde-4f2d-11e6-842f-17c5bbf40e9d.jpg)
I thought this was because of lg instead of sm but might not be so.